Java/E-montazysta/src/main/java/com/emontazysta/model/Orders.java

```diff
@@ -39,4 +39,12 @@ public class Orders {
     @JsonBackReference
     @ManyToOne
     private Company company;
+
+    @JsonBackReference
+    @ManyToOne
+    private Manager manager;
+
+    @JsonBackReference
+    @ManyToOne
+    private Foreman foreman;
```

Comment:
foreman -> assignedTo

Missing association with Specialist(filledBy?) and SalesRepresentative (createdBy?)